aruco_ros/src/aruco_detector.cpp

```diff
@@ -207,19 +210,120 @@ class ArucoSimple
     return "Unknown code";
   }
 
+  int get_home_from_id(int id){
+    char str[100];
+
+    for (unsigned i = 0; i < num_markers_in_list; ++i){
+      if (marker_id[i] == id){
+        return i;
```

Comment:
The first position in the list is the marker 500 which is technically not supported anymore, good home location are between 1 and 10, so 0 corresponds to an unknown home location. (either 500 or not recognized code)